src/Controller/Component/ToolbarComponent.php

```diff
@@ -39,8 +39,7 @@ public function __construct(ComponentRegistry $registry, $settings = [])
     {
         parent::__construct($registry, $settings);
 
-        $msg = 'DebugKit is now loaded through plugin bootstrapping. Make sure you have ' .
-            '`Plugin::load("DebugKit", ["bootstrap" => true]);` in your application\'s bootstrap.php.';
+        $msg = __d('debug_kit', 'DebugKit is now loaded through plugin bootstrapping. Make sure you have `Plugin::load("DebugKit", ["bootstrap" => true]);` in your application\'s bootstrap.php.');
```

Comment:
This is a really bad idea.
If you start using translations, never ever do that with code pieces or actual literal words that must not be translated, that includes the plugin name here, as well as the the complete code piece.

So use the {0} {1} ... placeholders then.

For the record: I still think it would be wiser to completly remove the translations.